docs/json_rpc_api.mediawiki

```diff
@@ -322,6 +322,10 @@ the method name for further details such as parameter and return information.
 |N
 |Returns the current value of all locked funds in the ticket pool.
 |-
+|[[#gettreasuryspendvotes|gettreasuryspendvotes]]
+|N
```

Comment:
I wonder if this should be made available to limited users.  I think so.  Obviously the map would need to be updated accordingly, but seeing this brings up the question.

What do you guys think?

Reply:
Seems reasonable, though this call might involve backtracking a large(ish) number of blocks to account for all votes of multiple tspends.

I'm not sure what the threat model or limitations are with regards to the limited user, so I'm not sure whether that's a problem or not.

Reply:
The general idea of limited users is that if you were to expose an RPC publicly, they shouldn't be able to really do anything that could seriously harm and/or shutdown the process.

I suppose it's fine to prevent limited users from using this RPC for now since it is incredibly poorly optimized (not due to the fault of this PR, but due to the fact there is no proper index being maintained for it).

Reply:
I am torn but I think I am leaning limited users as well. The only reason to open this up is for SPV wallets methinks. I say we keep it limited for now and open it up later if needed.

Reply:
Alright.  Limited it is.